manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -58,8 +58,11 @@ def readConfig(self):
         self.cert_file = os.path.join(consumerCertDir, cert)
         self.key_file = os.path.join(consumerCertDir, key)
 
-    def _connect(self, rhsm_username=None, rhsm_password=None):
+    def _connect(self, config = None):
         """ Connect to the subscription-manager. """
+
+        # Do we want to treat the rhsm_config as the default, and everything else as overrides, or
+        # should we only take from one set of configuration?
```

Comment:
I would say that these option should override options from rhsm_config. So use rhsm_* option from virt-who config if defined and if not, use value from rhsm.conf.